### ISSUE TITLE
Undefined chars in font throw exceptions

### ISSUE THREAD

**Issue by looksystems** [DanieLeeuwner/JSZPL#5]
I didn't realise but there's only a limited character set in the font definitions.

Attempting to print a string which contains characters defined in B64fonts will throw error in Text.calculateSize:

`
width += character[0].length
`
as character is undefined.

Here's a naive workaround, which assumes zero width:

`
width += typeof(character) !== 'undefined' ? character[0].length : 0
`

Arguably better to use width of space character.

Best solution is to have fully character set defined in the font, of course.

Hope this helps someone else.

**Comment by looksystems** [comment]
Any chance of incorporating this fix?

**Comment by DanieLeeuwner** [comment]
![empty_characters](https://user-images.githubusercontent.com/16599761/58323876-89a9b100-7e25-11e9-88cc-ea682e684ff9.PNG)

Created fix in f2190e2. Will merge into master soon.

**Comment by Archymede123** [comment]
Hello, thanks for the fix and the lib! 
Is it on purpose that it hasn't been published to npm / yarn ? 

If anyone sees this issue, I've ran the following command to get the fix : 
```
yarn add DanieLeeuwner/JSZPL#f2190e28a83dc5648b2dc438409cc2d39256a6dc
```
